src/app/components/Search/ResultsMetadata.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export const ResultsMetadata = (metadata) => {
+  if (metadata.metadata.total === 0) {
+    return (
+      <div className="nypl-results-summary" aria-live="assertive" aria-atomic="true" role="presentation">
+        Your search yielded no results. Please try again.
+      </div>
+    );
+  }
+
+  return (
+    <div className="nypl-results-summary" aria-live="assertive" aria-atomic="true" role="presentation">
```

Comment:
Does this work with screenreaders? You can try always rendering an empty div with the `aria-live` attribute and just updating the text based on `metadata.metadata.total`.

Reply:
just added this from the design toolkit recommendations. I'm sure that's not necessarily the way to go and all of this may change when UX gets more involved.

Reply:
Cool. I think leaving it as is is fine.